### ISSUE TITLE
Expansion of the pH SENSOR_MODEL names, with potential firmware version specification

### ISSUE THREAD

**Issue by tlmaurer** [nvs-vocabs/R27#13]
Hello,

The current SENSOR_MODEL definitions for pH sensors on BGC-Argo floats are currently limited to "DURA" (MBARI) and "SEAFET" (SEABIRD) within the Argo system.  Both MBARI and SBS have upgraded pH sensor types in development (Gasket DuraFET, and pressure-compensated pH sensor, respectively), with a limited number already deployed on floats in the testing phase.  There is thus a need to expand the pH SENSOR_MODEL vocabulary to include these newer versions.  It has been suggested that the latest MBARI sensor type be labelled "DURAV2", while SBS has identified their latest model as "SBS-58".  Considering the discussion on sensor firmware version (https://github.com/nvs-vocabs/R27/issues/8)  and @apswong latest comment, DURAV2 might be best left as "DURA", with an alternate SENSOR_FIRMWARE_VERSION specification, compared to the pre-existing DURA model.  Thoughts?

**Comment by tlmaurer** [comment]
This requires further review/discussion.  There was at one point interest in MBARI's newer model to be named GDF (or remain as DURA, with further SENSOR_FIRMWARE_VERSION specification).

Additionally, considering the current status of parts acquisition for pH sensors, it is unclear if / how many SBS-58 will be available in the future.

**Comment by tcarval** [comment]
> I'd like to officially propose the addition of a new pH SENSOR_MODEL = "GDF", with aligned SENSOR_MAKER = "MBARI".

Can you confirm that this is a new sensor, not a renaming of MBARI sensors ?
Is there a link to the sensor documentation ?

**Comment by tlmaurer** [comment]
Yes, this is a new sensor design (similar root components, same sensor output and processing chain).

**Comment by tcarval** [comment]
OK, I added this new entry in reference table 27:
GDF | MBARI | New pH sensor design : similar root components (DURA), same sensor output and processing chain | TRANSISTOR_PH | DURAFET | 07/12/2022 | added by t. carval for t. maurer 

The change is implemented in the format checker : https://github.com/euroargodev/ArgoNetCDF/releases/tag/2.7.10

**Comment by tcarval** [comment]
To do : update NVS table 27
